### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Louis Haye

### DISCUSSION THREAD

**Discussion by Artotrogo** [heig-vd-dai-course/heig-vd-dai-course#258] [Show and tell]
https://github.com/Artotrogo/Artotrogo/blob/main/README.md

**Comment by ludelafo** [comment]
Simple, yet effective profile, thank you!

Have you been working at Romande Energie for a long time? :)

**Reply by Artotrogo** [reply]
Not really, I started 2-3 month ago ! I was supposed to work there for only 2 months, but they were very pleased with my work so they signed me in for another year (renewable) 👍

**Reply by ludelafo** [reply]
That's nice to hear, I'm happy for you! If you know Guillaume Schranz from Romande Energie, he's a friend of mine!

**Reply by Artotrogo** [reply]
Thank you ! 
Of course, everybody knows Guillaume ! Haha
But yeah, we work in the same team !

**Reply by ludelafo** [reply]
Hahah such a nice and social guy!
